randovania/data/json_data/prime2.txt

```diff
@@ -653,22 +688,20 @@ Asset id: 3997643454
                   Difficulty: Normal
                   Any of the following:
                       Light Beam and Slope Jump (Normal)
-                      Morph Ball and Morph Ball Bomb
+                      Morph Ball and Morph Ball Bomb and Bomb Jump (Normal)
   > Path of Eyes/Event - Path of Eyes Light Block to Windchamber Gateway
       All of the following:
           Light Beam
           Any of the following:
-              Charge Beam and Space Jump Boots
-              Charge Beam and Slope Jump (Normal) and Difficulty: Normal
-              Space Jump Boots and Light Ammo
-              Light Ammo and Slope Jump (Normal) and Difficulty: Normal
+              Charge Beam or Space Jump Boots or Light Ammo
```

Comment:
This seems wrong. Moving the block expects you to be able to shoot Light Beam, so "Light Beam and (Charge or Light Ammo)" must be always true.

Reply:
Yeah that doesn't look correct. I'll have another look at it.